### PR TITLE
feat(Page Header) adding base implementation

### PR DESCRIPTION
**Pull request by DusanMilko**
Adding base implementation of page header

Supports:
- Title
- Subtitle
- Breadcrumbs
- White-labeling (custom background color)
- Surfaced details - next to title - open ended
- Actions - open ended
- Contained width

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
 // export directly from the index file to work around some bugs with
 // typescript and ngc and "barrel" files - https://github.com/ng-packagr/ng-packagr/issues/195
-export * from './page-header/index';
+export { PageHeaderComponent, PageHeaderModule } from './page-header/base/index';
 export * from './table/index';
 
 // Org specific components
-export * from './sterling/index';
+export {
+	PageHeaderComponent as SterlingPageHeaderComponent,
```
Comment (cal-smith) [inline src/index.ts:8]: I think we can safely rename the component classes at the source in this case

Comment (DusanMilko) [reply src/index.ts]: Updated

**Comment by cal-smith** [comment]
I don't know if this will trigger a release ... generally we need at least one commit with a `feat(thing): description` or `fix(thing): description` for a release to trigger ... I'll try amending the merge commit and see if that does it, otherwise I'll push a `feat`

**Pull request merged**